config/environments/development.rb

```diff
@@ -15,8 +15,11 @@
   # Enable/disable caching. By default caching is disabled.
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
-
-    config.cache_store = :memory_store
+    config.cache_store = :redis_store, 'redis://redis:6379/0/cache',
+                         {
+                           # Needed for redis to evict keys in volatile-lru mode
+                           expires_in: 30.days
```

Comment:
I'm still wondering if redis is the right call for this cache layer. Do we have a good reason to expire the entry after 30 days?